### PR TITLE
Release Testing Guide proposal

### PR DESCRIPTION
**Pull request by it33**
Proposing changes to enable community testing of Mattermost Desktop app: 

- Adding documentation for Windows functionality 
- Adding guide for community members to help test releases
- Rename "menubar" to "Menu Bar" 

Based [on Android App Testing guide](https://github.com/mattermost/android/blob/master/TESTING.md). 

I'd propose if this PR is accepted to in future: 

- Open [Help Wanted] issues for community members to help test Windows 8 and Windows 10
- Document Mac functionality and testing guides, and post [Help Wanted] issues as well
- Decide which Linux platforms are supported and add documentation and testing guides

### CODE REVIEW

File: TESTING.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+To contribute to the process of testing the Mattermost Desktop App: 
+
+1. If you haven't already, create an account on https://pre-release.mattermost.com/core
+  - Set your username to be the same as your GitHub username
+2. Follow the testing guide for your platform
+  - [Windows Testing Guide](#windows-testing-guide)
+3. Go to the [Public Test Channel](https://pre-release.mattermost.com/core/channels/public-test-channel) and try the following: 
+  1. Post the message with information on what you're testing, for example: `Testing Mattermost Desktop App 1.07 on Windows 7 SP1 64-bit`
+  2. Reply to the post by clicking on "**...**" then "**Reply**" with `This is a comment including files` and upload five (5) files including at least one image, one sound file and one video clip from your Android device.
```
Comment (yuya-oc) [inline TESTING.md]: This should be "PC" or "computer".

File: TESTING.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+Your testing contribution, including GitHub username, will be listed under the [Verified Operating Systems](TESTING.md#verified-operation-systems) section of this document.
+
+## Verified Operating Systems 
+
+The following chart summarizes the operating systems which the Mattermost Desktop App releases have been tested:
+
+| Version | OS | Issues | Tester | Date |
+|:--- |:--- |:--- |:--- |:--- |
+| 1.07 | Windows 7 SP1 64-bit | [#63](https://github.com/mattermost/desktop/issues/63) | [@it33](https://github.com/it33) | 2016-03-12 |
```
Comment (yuya-oc) [inline TESTING.md]: Could you use "1.0.7"?

**Comment by it33** [comment]
@yuya-oc, thanks for the feedback, corrected now.

**Comment by yuya-oc** [comment]
Thank you! I will follow in next release.

**Pull request merged**